### PR TITLE
doc: changing some  css and adding background color

### PR DESCRIPTION
**Pull request by mariyasada**
-changing some css 
-added comment section, logged in user can comment

### CODE REVIEW

**Review by Bhakti199 — APPROVED** [review]
This PR looks good to me. 
Background color seems good. 
This PR is good to merge.

File: src/Components/Hamburger/Hamburger.jsx [hunk 1]
```diff
@@ -1,40 +1,107 @@
...
+        <NavLink
+          to="/history"
+          className="hamburger-menu-item"
+          style={getActiveStyle}
+        >
+          <li className="hamburger-item flex-center">
+            <MdHistory />
+            History
+          </li>
+        </NavLink>
```
Comment (JantuDeb) [inline src/Components/Hamburger/Hamburger.jsx:82]: store these in array then loop through it to render the `NavLink`

**Review by JantuDeb — APPROVED** [review]
This is looking good

File: src/Components/Hamburger/Hamburger.jsx (hunk 1 above)
Comment (mariyasada) [reply src/Components/Hamburger/Hamburger.jsx:82]: will do

**Pull request merged**